### PR TITLE
re-do UI for verify-to-join

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ one more time

### CODE REVIEW

**Comment by camhashemi** [comment]
screen shot 2015-11-11 at 4 17 34 pm

File: kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html [hunk 1]
```diff
@@ -96,8 +97,15 @@
           </tr>
         </table>
       </div>
-      <button class="kf-pw-org-inv-btn kf-pw-org-inv-btn-accept" ng-if="org.pending && org.notDeclined" ng-click="acceptInvite(org)">Accept</button>
-      <button class="kf-pw-org-inv-btn kf-pw-org-inv-btn-decline" ng-if="org.pending && org.notDeclined" ng-click="declineInvite(org)">Decline</button>
+      <button class="kf-pw-org-btn kf-pw-org-btn-accept" ng-if="org.pending && !org.declined" ng-click="acceptInvite(org)">Accept</button>
+      <button class="kf-pw-org-btn kf-pw-org-btn-decline" ng-if="org.pending && !org.declined" ng-click="declineInvite(org)">Decline</button>
+      <div class="kf-pw-org-email" ng-if="org.potential && org.viewer.emails && me.experiments.indexOf('admin') !== -1">
+        <div ng-repeat="email in org.viewer.emails">
+          <p class="kf-pw-org-email-txt">{{ email }}</p>
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html]: Small note: I'd like to transition our code from  `<tag>{{ blah }}</tag>` bindings to using `<tag ng-bind="blah"></tag>`. On slow-rendering browsers, users might see the `{{ blah }}` which does not look polished, so I'll be calling these out in code review when I see them.

**Comment by cvializ** [comment]
daft punk one more time

**Comment by cvializ** [comment]
:+1: hooray!

File: kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html]: cool, noted! thanks

**Pull request merged**